elasticsearch/files/elasticsearch.yml

```diff
@@ -41,7 +41,9 @@ cluster.name: {{ server.cluster.name }}
 # from configuring them manually. You can tie this node to a specific name:
 #
 # node.name: "Franz Kafka"
-node.name: {{ server.get('name', '${HOSTNAME}') }}
+{%- if server.name is defined %}
+node.name: {{ server.name }}
```

Comment:
The thing is that when I deploy elasticsearch currently with our model I have in the elasticsearch configuration file something like:

```
node.name: ${HOSTNAME}
```